core/keyboard_nav/marker.js

```diff
@@ -11,19 +11,19 @@
  */
 'use strict';
 
-goog.provide('Blockly.Marker');
+goog.module('Blockly.Marker');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.ASTNode');
-
-goog.requireType('Blockly.blockRendering.MarkerSvg');
+const ASTNode = goog.require('Blockly.ASTNode');
```

Comment:
Lint says ASTNode and MarkerSvg are never used. I think both can become `requireType` calls instead